### PR TITLE
Update Tailwind config to match typography from Figma

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description

This PR configures Tailwind to provide the typographical utilities as defined in [Figma](https://www.figma.com/file/vEoItpMQyFZb5AZV9aoP7I/Audio-integration?node-id=1147%3A11088). These utilities will be useful in all UI building PRs, hence this is a bit urgent.

## Technical details

The line-heights are now included in the font-sizes based on the defaults. They can be configured on a per-usage basis using the `leading-*` utilities, also configured in the PR itself.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by zackkrida — APPROVED** [review]
This looks really nice!

File: tailwind.config.js [hunk 1]
```diff
@@ -66,19 +66,58 @@ module.exports = {
...
       // Headings, where h[n] => [7-n]xl
-      '1xl': '1.12rem',
-      '2xl': '1.25rem',
-      '3xl': '1.43rem',
-      '4xl': '1.75rem',
-      '5xl': '2.25rem',
-      '6xl': '3.56rem',
+      '6xl': [
+        '2.2857142857rem', // 32px
+        '1.3', // 130%
```
Comment (sarayourfriend) [inline tailwind.config.js:76]: What does the second number in the array do? I couldn't find information about it the tailwind configuration docs.

Edit: Oh, is it line-height or something?

**Review by sarayourfriend — COMMENTED** [review]
LGTM! 🚀

File: tailwind.config.js (hunk 1 above)
Comment (zackkrida) [reply tailwind.config.js:76]: yep! line height.

Comment (dhruvkb) [reply tailwind.config.js:76]: For posterity, here is the [relevant documentation](https://tailwindcss.com/docs/font-size#providing-a-default-line-height).

**Pull request merged**